### DISCUSSION TITLE
[Fixed: Can now be sorted with ```Top:All```] Sort by upvotes by default

### DISCUSSION THREAD

**Discussion by Josh65-2201** [community/community#5458] [Discussions]
Currently it seems to sort by reply's and suggested answers. It should really be sorted by up votes by default so that its make sense on what people want. Then there could be an option to lock the upvotes so that discussions don't get locked at the top. A sorting option for reply's could them be added.

**Comment by rowlandm** [comment]
I agree

**Comment by markseuffert** [comment]
See also #3328
